improver_tests/calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -385,6 +408,146 @@ def test_realizations_predictor_max_iterations(self):
             result, self.expected_realizations_coefficients
         )
 
+    @ManageWarnings(
+        ignored_messages=[
+            "Collapsing a non-contiguous coordinate.",
+            "Minimisation did not result in convergence",
+            "divide by zero encountered in",
+        ],
+        warning_types=[UserWarning, UserWarning, RuntimeWarning],
+    )
+    def test_mean_predictor_each_point(self):
+        """
+        Test that the expected coefficients are generated when the ensemble
+        mean is the predictor for a normal distribution and coefficients are
+        calculated independently at each grid point. The coefficients are
+        calculated by minimising the CRPS and using a set default value for the
+        initial guess.
```

Comment:
Does the CRPS minimiser know anything about whether the initial guesses are default values or have been estimated using linear regression? I'm pretty sure it doesn't care, and so it is unimportant to this test whether the initial values are default or estimated as far as I can tell (you talk about default values in the tests below as well).

There is potential confusion with this comment in concert with the shared variable name `each_point` (as commented on earlier), which in this context has no relation to whether the initial guess is a default value or a linear fit estimate.

Reply:
Yes, you're correct in that the minimisation doesn't know anything about where the initial guess has come from. I've removed the reference to the initial guess in these tests.